components/kyma-environment-broker/internal/hyperscaler/azure/client.go

```diff
@@ -19,12 +28,26 @@ type AzureClient struct {
 	// the actual azure client
 	eventhubNamespaceClient eventhub.NamespacesClient
 	resourcegroupClient     resources.GroupsClient
+	logger                  logrus.FieldLogger
+}
+
+type ResourceGroupDoesNotExist struct {
+	S string
```

Comment:
Why did you choose `S`? `S` is errorMessage everywhere. I think we can make it explicit.